CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributor Strategy Special Interest Group Contributing Guide
+*this is a bootstrapped document that will update frequently in the next few
+months as we lay the groundwork for the operations of the SIG. Now onto the
+show...:*
+
+🛬Thanks for landing here! There's a good chance you want to help the
+contributor side of the code in one of our working groups, get involved on a
+regular basis, or simply be curious, no matter your reason are excited to have
+you.
+
+This [SIG] (Special Interest Group) is an extension of the [TOC] that works to
+build programs and guidance to help scale and grow open source communities. We
+work in two CNCF GitHub repositories:  
+🆕[cncf/sig-contributor-strategy]  
+ our meta docs that cover our governance, how we operate, and resources that we
+ collect along the way.  
+🔜[cncf/contribute]  
+will house contributing information and guidance that we provide to CNCF
+projects for contributor strategy topics. The intention is to grow this as a
+resource for all contributors - aspiring, current, and maintainers.  
+
+Our [charter] describes our mission and initial scope of what we’d like to
+accomplish. Each of our members are closely involved with at least one CNCF
+project; we are here to learn as much as those we serve.
+
+## Ground Rules
+We very much welcome everyone on any of our [communication channels] including
+through GitHub Issues and Pull Requests (PRs) and ask all contributors to
+maintain our welcoming culture. We follow the [CNCF’s code of conduct].
+
+Async communication is our preference because of our global nature but
+absolutely value face to face time and meetings when it makes sense. Slack works
+well for real time questions and discussion but we will move decisions and key
+tasks to GitHub or the mailing list. Our communication channels are on our
+[README].
+
+We don’t sponsor specific company branded documentation and resources as
+official guidance to contributors and/or projects. A resources list[link tbu]
+that may link to other sources that are not official, including company branded
+information, in the strategy repo, is strictly for our groups planning purposes
+only.
+
+## How to Help
+
+**Contributors and/or Maintainers of a CNCF incubated or graduated project:**  
+- Be a liaison for your project with our group
+  - Why? The ecosystem is very large, if you stay connected, you’ll be closer to
+resources and advice.
+  - Take back this information to build (or continue to build!) intentional
+community and governance spaces and/or roles in your project to do this work at
+an operational level
+- Join us for a Contributor Strategy AMA on the 4th Thursday meeting of each month
+ at the last :30 minutes as you need. This is your time to ask us general or
```

Comment:
Suggestion instead: "on the 4th Thursday every month at 10am Pacific / 1700 UTC".  Any incarnation of "the last 30 min of the meeting" is going to be hard to parse.

Reply:
I was trying to manage that we will need time to do general sig discussions and triage. what about: "Join us for a Contributor Strategy AMA on the 4th Thursday meeting of each month. We'll do a general SIG update and discussion in the beginning and then the floor is yours to ask us anything....[finish the statement]"

Reply:
sure, that works